### PR TITLE
Factor out getting collaborators (GithubProject)

### LINKED ISSUE #224: Identical implementation for collaborators (Github Project)
In `GithubProject` class there is identical implementation for `who_can_close_issue` and `who_can_merge_pr`, is it intentional? Or can we factor it out since it's identical?

### PR DESCRIPTION
**Pull request by mfocko**
Closes #224

### CODE REVIEW

File: ogr/services/github.py [hunk 1]
```diff
@@ -366,21 +366,11 @@ def who_can_close_issue(self) -> Set[str]:
...
-                f"Current Github token must have push access to view repository permissions."
-            )
-            return set()
-
-        usernames = []
-        for login, permission in collaborators.items():
-            if permission in ["admin", "write"]:
-                usernames.append(login)
+    def who_can_close_issue(self) -> Set[str]:
+        return self.__collaborators()
```
Comment (lachmanfrantisek) [inline ogr/services/github.py]: The name can be a little bit confusing -- I think that no all collaborators have that permission. (Also you can use some verb in the name like `get`.)

Comment (mfocko) [reply ogr/services/github.py]: it's given by interface... another refactor?

Comment (lachmanfrantisek) [reply ogr/services/github.py]: I mean the `__collaborators`.

Comment (mfocko) [reply ogr/services/github.py]: oh ok :D

**Pull request merged**